daprdocs/content/en/developing-applications/building-blocks/actors/actor-reminders-partitions.md

```diff
@@ -0,0 +1,98 @@
+weight: 30
+description: Scale actor reminders storage via partitioning
+---
+
+{{% alert title="Preview feature" color="warning" %}}
+Actor reminders partitioning is currently in [preview]({{< ref preview-features.md >}}).
+{{% /alert %}}
+
+## Actor reminders
+An Actor reminder enables scheduled events to be triggered on a given virtual actor. Actor reminders are persisted and continue to be triggered after sidecar restarts. Prior to Dapr runtime version 1.3, reminders were persisted on a single record in the actor state store:
+
+| Key      | Value |
+| ----------- | ----------- |
+| `actors\|\|<actor type>` | `[ <reminder 1>, <reminder 2>, ... , <reminder n> ]` |
+
+Applications that register many reminders can experience the following issues:
+
+* Low throughput on reminders registration and deregistration
+* Limit on total number of reminders registered based on the single record size limit on the state store
+
+Since version 1.3, applications can now enable partitioning of actor reminders in the state store. As data is distributed in multiple keys in the state store. First, there is a metadata record in `actors\|\|<actor type>\|\|metadata` that is used to store persisted configuration for a given actor type. Then, there are multiple records that stores subsets of the reminders for the same actor type.
+
+| Key      | Value |
+| ----------- | ----------- |
+| `actors\|\|<actor type>\|\|metadata` | `{ "id": <actor metadata identifier>, "actorRemindersMetadata": { "partitionCount": <number of partitions for reminders> } }` |
+| `actors\|\|<actor type>\|\|<actor metadata identifier>\|\|reminders\|\|1` | `[ <reminder 1-1>, <reminder 1-2>, ... , <reminder 1-n> ]` |
+| `actors\|\|<actor type>\|\|<actor metadata identifier>\|\|reminders\|\|2` | `[ <reminder 1-1>, <reminder 1-2>, ... , <reminder 1-m> ]` |
+| ... | ... |
+
+If the number of partitions is not enough, it can be changed and Dapr's sidecar will automatically redistribute the reminders's set.
+
+## Enabling actor reminders partitioning
+Actor reminders partitioning is currently in preview, so enabling it is a two step process.
+
+### Preview feature configuration
+Before using reminders partitioning, actor type metadata must be enabled in Dapr. For more information on preview configurations, see [the full guide on opting into preview features in Dapr]({{< ref preview-features.md >}}). Below is an example of the configuration:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: myconfig
+spec:
+  features:
+    - name: Actor.TypeMetadata
+      enabled: true
+```
+
+### Actor runtime configuration
+Once actor type metadata is enabled as an opt-in preview feature, the actor runtime must also provide the appropriate configuration to partition actor reminders. This is done by the actor's endpoint for `GET /dapr/config`, similar to other actor configuration elements. Here is a snipet of an actor written in Golang providing the configuration:
+
+```go
```

Comment:
Include code for .NET, Java, Python. See https://github.com/dapr/docs/blob/v1.3/daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-services-grpc.md#step-2-invoke-the-service as an example using code TABS

Reply:
done.